tests/testthat/test-fill.R

```diff
@@ -0,0 +1,72 @@
+test_that("all missings left unchanged", {
+  tbl <- tibble(
+    int = c(NA_integer_, NA),
+    dbl = c(NA_real_, NA),
+  )
+  dt <- lazy_dt(tbl, "DT")
+
+  down <- fill(dt, int, dbl)
+  up <- fill(dt, int, dbl, .direction = "up")
+
+  expect_identical(collect(down), tbl)
+  expect_identical(collect(up), tbl)
+})
```

Comment:
I think you can remove this, since it's mostly testing the behaviour of `nafill()`?